packages/extension-chakra-storefront/jest-setup.js

```diff
@@ -137,6 +137,7 @@ jest.mock('@salesforce/cc-datacloud-typescript', () => ({
 // from nodejs util in testing environment.
 global.TextEncoder = require('util').TextEncoder
 global.TextDecoder = require('util').TextDecoder
+global.structuredClone = (v) => JSON.parse(JSON.stringify(v))
```

Comment:
I ran into this problem during the fixes. The solution is to add a mock for it.  See [here](https://stackoverflow.com/questions/73607410/referenceerror-structuredclone-is-not-defined-using-jest-with-nodejs-typesc)